### ISSUE TITLE
current master failed to start on Debian 

### ISSUE THREAD

**Issue by gwh-cpnet** [Nefelim4ag/systemd-swap#107]
I just checkout the latest master, built, and installed it on a new Debian host. Here is the log I got. Maybe there is something wrong with the lately code cleanup. 

```
sudo journalctl -u systemd-swap.service
-- Logs begin at Tue 2020-06-02 02:31:53 UTC, end at Tue 2020-06-02 02:42:05 UTC. --
Jun 02 02:31:55 vm systemd[1]: Started Manage swap spaces on zram, files and partitions..
Jun 02 02:31:55 vm systemd-swap[393]: /usr/bin/systemd-swap: line 538: syntax error: unexpected end of file
Jun 02 02:31:55 vm systemd[1]: systemd-swap.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Jun 02 02:31:55 vm systemd[1]: systemd-swap.service: Failed with result 'exit-code'.
Jun 02 02:31:55 vm systemd[1]: systemd-swap.service: Consumed 5ms CPU time.
```

earlier checkout works fine.

**Comment by Jarel1337** [comment]
Copy paste from other (closed) issue:
#104 Fixes this, you can apply it yourself whilst waiting for it to get upstreamed.

**Comment by Jarel1337** [comment]
Fixed in 884f939a75a725bca14973073a629b27cdee323f.
